static/src/js/App.jsx

```diff
@@ -276,22 +289,24 @@ const App = () => {
   return (
     <ErrorBoundary>
       <Provider store={store}>
-        <ToastProvider ref={window.reactToastProvider}>
-          <Helmet
-            defaultTitle="Earthdata Search"
-            titleTemplate={`${titleEnv} %s | Earthdata Search`}
-          >
-            <meta name="description" content={description} />
-            <meta property="og:type" content="website" />
-            <meta property="og:title" content={title} />
-            <meta property="og:description" content={description} />
-            <meta property="og:url" content={url} />
-            <meta property="og:image" content={ogImage} />
-            <meta name="theme-color" content="#191a1b" />
-            <link rel="canonical" href={url} />
-          </Helmet>
-          <RouterProvider router={browserRouter} />
-        </ToastProvider>
+        <GraphQlProvider>
```

Comment:
Should GraphQlProvider be in the AppLayout instead?

Reply:
Kinda torn on this. Seems like AppLayout is more concerned with containers that are connecting to state. I saw this is where we had put other providers, which seemed more similar.